rust/rope/src/delta.rs

```diff
@@ -264,6 +264,17 @@ impl<N: NodeInfo> Delta<N> {
         )
     }
 
+    /// Returns the sum length of the inserts of the delta.
+    pub fn inserts_len(&self) -> usize {
+        self.els.as_slice().iter()
```

Comment:
The `as_slice` doesn't seem to be needed here, you should be able to do `iter` directly on a Vec with the same effect.